cfme/tests/containers/test_ssl_providers.py

```diff
@@ -139,7 +140,7 @@ def test_add_hawkular_provider_ssl(provider, appliance, test_item, soft_assert,
     except AssertionError:
         soft_assert(False, provider.name + ' wasn\'t added successfully using ' +
                     test_item.default_sec_protocol + ' security protocol and ' +
-                    test_item.hawkular_sec_protocol + ' hawkular security protocol')
+                    test_item.hawkular_sec_protocol + ' metrics security protocol')
```

Comment:
**REQUIRED**
Please use `format` to concatenate strings.

Reply:
Done